### DISCUSSION TITLE
GTK4 and UI

### DISCUSSION THREAD

**Discussion by Mlocik97** [wwmm/easyeffects#858] [General]
Hello,

GTK4 is out, so I want to just talk about it (and relating parts to PulseEffects), and maybe new idea about UI/UX for future.
I think, now, we don't use space on display effectively, so it would be propably nice in some future rearange layout.

**Comment by wwmm** [comment]
I haven't followed GTK4 development. I still have to see what new things it is bringing for layouts.

**Comment by wwmm** [comment]
GTK4 branch https://github.com/wwmm/pulseeffects/tree/gtk4. It is far from finished.  So far there is only the main window and the presets menu.

**Reply by wwmm** [reply]
As I am not sure most people are aware of this github beta feature I decided to also open an issue about the port and pin it https://github.com/wwmm/pulseeffects/issues/904
